FreeRTOS/FreeRTOS-Plus-TCP
# Build separation for ARP/ND

Improve Build separation for IPv4/IPv6

Description
-----------
Only include FreeRTOS_ARP if IPv4 is enabled. Move everything else into routing which calls either ARP or ND. Also create a frame agnostic helper for checking the cache.

Test Steps
-----------
Need to finish adjusting a couple tests

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
